.codecheckignore

```diff
@@ -35,4 +35,3 @@
 ^UNITTESTS
 ^storage/blockdevice/tests/UNITTESTS
 ^storage/kvstore/tests/UNITTESTS
-^drivers/tests/UNITTESTS
```

Comment:
Just an observation: There seem to be no style changes needed in this commit. The style checking CI is passing. If there were files out of style in drivers/tests/UNITTESTS, this commit would have needed fixes to get those files back in style.

Reply:
Thanks Jaeden, noted. I did run astyle on them but they turned out to all be fine, luckily.